airflow/plugins_manager.py

```diff
@@ -184,13 +184,23 @@ def is_valid_plugin(plugin_obj):
     return False
 
 
+def register_plugin(plugin_instance):
+    """
+    Start plugin load and register it after success initialization
+
+    :param plugin_instance: subclass of AirflowPlugin
+    """
+    global plugins  # pylint: disable=global-statement
+    plugin_instance.on_load()
```

Comment:
We don't need to check if `on_load` is present since we already verify the plugin is a subclass of `AirflowPlugin`.